Tests/LibWeb/Text/expected/wpt-import/accname/name/shadowdom/basic.txt

```diff
@@ -6,8 +6,7 @@ Rerun
 
 Found 2 tests
 
-1 Pass
-1 Fail
+2 Pass
 Details
-Result	Test Name	MessageFail	aria-labelledby reference to element with text content inside shadow DOM	
+Result	Test Name	MessagePass	aria-labelledby reference to element with text content inside shadow DOM	
```

Comment:
I don't quite follow - shouldn't this test fail in earlier commits then? This change should ideally be included in the commit that changes its outcome.

Reply:
> I don't quite follow - shouldn't this test fail in earlier commits then? This change should ideally be included in the commit that changes its outcome.

OK yeah looking back at the commit history, it seems I actually regressed it _on this branch_, in https://github.com/LadybirdBrowser/ladybird/commit/0ef0cefc7fed58af3c133e28b4bef5218472c0b9#diff-674914bf5c766d81103cea8a5f43835e30ee7e0d33bd76820b5fcadc39fded96.

So I removed that mistaken expectations change from that commit — which in turn eliminates the subsequent commit I’d added here to fix that regression.